### ISSUE TITLE
Exclude options does not exclude some settings on "core-store"

### ISSUE THREAD

**Issue by SalahAdDin** [boazpoolman/strapi-plugin-config-sync#10]
Hi,

We are testing this plugin in order to synchronize our different environment settings.

We just want to persist the content type admin panel's configuration, so we want to exclude the others:
```javascript
"config-sync": {
    destination: "extensions/config-sync/files/",
    minify: false,
    importOnBootstrap: false,
    include: ["core-store", "role-permissions"],
    exclude: [
      "core-store.model_def_application",
      "core-store.model_def_items",
      "core-store.model_def_plugins",
      "core-store.model_def_strapi",
      "core-store.plugin_users-permissions_grant",
    ],
  },
```

Unfortunately, it does not work:
![image](https://user-images.githubusercontent.com/5034215/119223759-d3f0c600-bb03-11eb-870f-861585d51dc3.png)

What's the problem here?

Is there any way to use regex patterns here? lets say: `core-store.model_def_*` in order to exclude all model definition settings.

Thanks.

**Comment by boazpoolman** [comment]
Hi @SalahAdDin,
Thank you for your issue submission!

This seems like a bug. The exclude setting should work properly.
I will take a look at this. 

Also I agree a regex pattern would be very usefull here.
I will also take a stab at implementing this.

**Comment by boazpoolman** [comment]
This is fixed in `v1.0.0-beta.1`.
You can now exclude configs by giving the setting part of the config name.
Like: `core-store.plugin_content_manager`
